packages/ember/tests/integration/multiple-app-test.js

```diff
@@ -4,17 +4,15 @@ import {
 } from 'internal-test-helpers';
 import { Application } from 'ember-application';
 import { Component } from 'ember-glimmer';
-import { jQuery } from 'ember-views';
 import { assign, getOwner } from 'ember-utils';
 import { resolve } from 'rsvp';
 
 moduleFor('View Integration', class extends ApplicationTestCase {
 
   constructor() {
-    jQuery('#qunit-fixture').html(`
-      <div id="one"></div>
-      <div id="two"></div>
-    `);
+    document.getElementById('qunit-fixture').innerHTML = 
```

Comment:
We have a specialized system in place for this (added in https://github.com/emberjs/ember.js/commit/ff9117a1d647c7bf098a44b881289d546b972fe7). You should be able to do:

```js
class extends ApplicationTestCase {
  get fixture() {
    return `<div id="one"></div><div id="two"></div>`;
  }
}
```